content/en/docs/reference/config/istio.mesh.v1alpha1/index.html

```diff
@@ -40,7 +40,7 @@ <h2 id="Tracing">Tracing</h2>
 <td><code>lightstep</code></td>
 <td><code><a href="#Tracing-Lightstep">Lightstep (oneof)</a></code></td>
 <td>
-<p>Use a LightStep tracer.</p>
+<p>Use a Lightstep tracer.</p>
```

Comment:
All of the changes in this file will be overriden by the ref doc update. Changes need to be made: 

WARNING: THIS IS AN AUTO-GENERATED FILE, DO NOT EDIT. PLEASE MODIFY THE ORIGINAL SOURCE IN THE 'https://github.com/istio/api' REPO
source_repo: https://github.com/istio/api